src/gvmd.c

```diff
@@ -1133,8 +1129,15 @@ update_nvt_cache_retry ()
         }
       else if (child_pid == 0)
         {
+          const char *osp_update_socket;
+          osp_update_socket = get_osp_vt_update_socket ();
           if (osp_update_socket)
             exit (update_nvt_cache_osp (osp_update_socket));
+          else
+            {
+              g_warning ("%s: No OSP VT update socket set", __FUNCTION__);
+              exit (-1);
```

Comment:
I think this should be EXIT_FAILURE because that's what we use everywhere else.

Reply:
I've changed this in bdc83ddc70159dd1b722369e19c72e59ff3f43a7.